src/Arcus.BackgroundJobs.CloudEvents/Arcus.BackgroundJobs.CloudEvents.csproj

```diff
@@ -17,8 +17,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Arcus.EventGrid" Version="3.0.0-preview-1" />
-    <PackageReference Include="Arcus.Messaging.Pumps.ServiceBus" Version="0.1.0-preview-2" />
-    <PackageReference Include="Microsoft.Azure.EventGrid" Version="3.2.0" />
+    <PackageReference Include="Arcus.EventGrid" Version="3.0.0" />
+    <PackageReference Include="Arcus.Messaging.Pumps.ServiceBus" Version="20200312.0.0-PR-81" />
```

Comment:
I'd suggest to ship a new preview version instead

Reply:
Yeah, we could do that. Then we can remove the MyGet sources from the project files too.